publiccode.yml

```
@@ -16,10 +16,21 @@ categories:
   - data-analytics
   - data-collection
   - data-visualization
+  - open-data
 maintenance:
   type: none
 legal:
-  license: Apache-2.0
+  license: 'Apache-2.0'
+  #repoOwner: 'City of Chicago'
+  #authorsFile: 'AUTHORS'
```

Direi di creare un `AUTHORS` file, con info del tipo:
```
Copyright (c) 2019 Presidenza del Consiglio dei Ministri
The version control system provides attribution for specific lines of code.
```
che ne dici?

---

In lavorazione in #32 ... Ma non mescolerei le informazioni sugli autori e sulla licenza, per quest'ultima fa fede il [file LICENSE](https://github.com/italia/dati-ckan-docker/blob/master/LICENSE) che comunque va verificato.

---

OK! Comunque se pianifichi di inserire il file `AUTHORS` allora toglierei il `#`
```suggestion
authorsFile: 'AUTHORS'
```